lib/arrow.cpp

```diff
@@ -1839,7 +1881,7 @@ namespace grnarrow {
 
     void add_column_string(const char *value, size_t value_length) {
       auto column_builder =
-        record_batch_builder_->GetFieldAs<arrow::StringBuilder>(
+        record_batch_builder_->GetFieldAs<arrow::StringDictionaryBuilder>(
```

Comment:
Should we also use dictionary for not referenced string column?

Reply:
Honestly, I'm wondering if we should target not referenced string columns.
When a user uses a not referenced column, he(she) probably expect values inserted to the column do not have so much duplicated values, but it depends on actual data...